docker/docker-compose-with-cassandra.yml

```diff
@@ -148,7 +148,7 @@ services:
       - ${HOME}/.datahub/plugins:/etc/datahub/plugins
 
   datahub-actions:
-    image: public.ecr.aws/datahub/acryl-datahub-actions:${ACTIONS_VERSION:-v0.0.1-beta.11}
+    image: public.ecr.aws/datahub/datahub-actions:${ACTIONS_VERSION:-v0.0.1-beta.11}
```

Comment:
any reason for using ecr one here?

Reply:
And the image tag should be as per the new ones, right?
https://github.com/acryldata/datahub-actions/releases

Reply:
good catch this should not be ecr

> > should be compatible
> 
> Have you tested this on local?

Yep - test this locally with same configs (env) we use with quickstart